src/lib.rs

```diff
@@ -73,6 +73,13 @@ impl Default for Args {
     }
 }
 
+pub fn path_to_module(current_file: &Path, module: &str) -> PathBuf {
```

Comment:
Do you see this being used somewhere else? Why is it in lib?

Reply:
```
$ rg path_to_module src/**
src/sectionizer.rs
3:use crate::path_to_module;
54:                    let use_file = path_to_module(&path, file);

src/lib.rs
76:pub fn path_to_module(current_file: &Path, module: &str) -> PathBuf {

src/compiler.rs
6:use crate::{Blob, Block, Op, Prog, RustFunction, Type, Value, path_to_module};
1783:                    let path = path_to_module(file, &name);
```